### PR TITLE
neoprep epps

### PR DESCRIPTION
**Pull request by Galithil**
This adds two epps and a way to get file contents through the lims object.

### CODE REVIEW

**Comment by Galithil** [comment]
Also, we can get index sequences through the api now.

**Comment by Galithil** [comment]
Epps have been tested & validated

File: scripts/read_neoprep_log.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+from genologics.epp import attach_file
+
+def read_log(lims, pid, logfile):
+    logger=setupLog(logfile)
+    pro=Process(lims, id=pid)
+    f=None
+    for out in pro.all_outputs():
+        if out.type == "ResultFile" and out.name == "NeoPrep Output Log File":
+            fid=out.files[0].id
+            f=lims.get_file_contents(id=fid)
```
Comment (remiolsen) [inline scripts/read_neoprep_log.py]: What does "f" stand for? I'm assuming it rhymes with luck.

Comment (Galithil) [inline scripts/read_neoprep_log.py]: :D could have, but this time it stands for file_contents. I should not use such short variables. give me a minute

**Comment by Galithil** [comment]
I added comments so I might be able to understand what it does in a year or so

File: scripts/write_neoprep_csv.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            except:
+                logger.error("Cannot find the reagent label of output analyte {0}".format(out.id))
+                return None
+            try:
+                matches=proto_pattern.search(inp.udf['Covaris Protocol'])
+                ins_size=matches.group(1)
+            except:
+                logger.error("Cannot find the insert size of analyte {0} ({1})".format(inp.id, sname))
+                ins_size='350'
+
```
Comment (remiolsen) [inline scripts/write_neoprep_csv.py]: I like the paranoia. :hushed:

Comment (Galithil) [inline scripts/write_neoprep_csv.py]: The day I trust user input will be a sad day

**Comment by remiolsen** [comment]
Great! I could complain about whitespace things, but can merge it you want.

**Comment by Galithil** [comment]
Yep, i'd like you to merge, pretty please :)

**Pull request merged**